src/midje/parsing/util/recognizing.clj

```diff
@@ -44,7 +44,15 @@
                 (symbol? (second form))
                 (expect-arrows (name (second form))))))
 
-(defn start-of-prerequisite-arrow-sequence? [form]
+(defmulti start-of-prerequisite-arrow-sequence? tree-variant)
```

Comment:
Kind of a bummer having to implement every predict twice - for a zippper loc and for a clojure form - but I see that is the current pattern of the codebase.